src/app/containers/Header/ScriptLink/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useContext } from 'react';
+import { compile } from 'path-to-regexp';
+import { useRouteMatch } from 'react-router-dom';
+import ScriptLink from '@bbc/psammead-script-link';
+import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
+import { UserContext } from '#contexts/UserContext';
+import { ServiceContext } from '#contexts/ServiceContext';
+
+export const getVariantHref = ({ path, params, variant }) =>
+  compile(path)(
```

Comment:
This assumes that the user is currently on a page matched by one of our defined routes. We could be more defensive and return a default value if params is empty.